.gitignore

```diff
@@ -34,4 +34,5 @@ Extensions/BitBucket/DownloadArtifactBitbucket/bin/*
 Extensions/BitBucket/DownloadArtifactBitbucket/obj/*
```

Comment:
addressed in commit [9487e13](https://github.com/mmikkelson/vsts-rm-extensions/commit/9487e13813526333b507035f9027f6eda02e6019)